src/test/java/com/openshift/internal/restclient/authorization/AuthorizationKindsIntegrationTest.java

```diff
@@ -0,0 +1,52 @@
+ /*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc. Distributed under license by Red Hat, Inc.
+ * All rights reserved. This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors: Red Hat, Inc.
+ ******************************************************************************/
+package com.openshift.internal.restclient.authorization;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.openshift.internal.restclient.IntegrationTestHelper;
+import com.openshift.restclient.IClient;
+import com.openshift.restclient.ResourceKind;
+import com.openshift.restclient.model.authorization.IRole;
+
+/**
+ * @author Jeff Cantrill
+ */
+public class AuthorizationKindsIntegrationTest {
+
+	private IClient client;
+	private IntegrationTestHelper helper = new IntegrationTestHelper();
+
+	@Before
+	public void setup () {
+		client = helper.createClientForBasicAuth();
+	}
+	
+	@Test
+	public void testListRolesAssumingClusterAdmin(){
```

Comment:
This test keeps failing on both my Win10 and my RHEL 7.2 VM. I'm using CDK 2.3 (OCP 3.3)